### ISSUE TITLE
Browse into symlinks

### ISSUE THREAD

**Issue by raffaem** [virtio-win/kvm-guest-drivers-windows#796]
**Is your feature request related to a problem? Please describe.**

virtiofs support only mounting 1 folder at a time.

I want to create symlinks into this folder, and browse into them.

So to have a central repository of shared folders (although I don't know how secure this would be ... it means that guest can access host's entire filesystem, right?)

**Describe the solution you'd like**
Browse into symlinks

**Describe alternatives you've considered**

Mounting more than one folder at a time (not currently possible).

Using SAMBA (would be too slow).

**Additional context**

Related #499

**Comment by viktor-prutyanov** [comment]
Hi

Host daemon (virtiofsd) prohibits guest to access files outside of shared folder (including access by symlink) because of security reasons.

**Comment by raffaem** [comment]
> Hi
> 
> Host daemon (virtiofsd) prohibits guest to access files outside of shared folder (including access by symlink) because of security reasons.

Oh ok, then I lost what #499 was about.

Is there a way to change that ans specify access to *some* of outside folders?

**Comment by viktor-prutyanov** [comment]
> Is there a way to change that ans specify access to _some_ of outside folders?

No, as far as I know.

**Comment by Daniel-Trevitz** [comment]
I think this is exacerbated by not being able to share / (ignoring the multiple drive topic).
What's weird though is that I can char /mnt.

Whenever I try to do so I get 
```
Error starting domain: internal error: qemu unexpectedly closed the monitor: 2022-07-14T11:33:48.736445Z qemu-system-x86_64: -device {"driver":"vhost-user-fs-pci","id":"fs0","chardev":"chr-vu-fs0","queue-size":1024,"tag":"daniel","bus":"pci.5","addr":"0x0"}: Failed to write msg. Wrote -1 instead of 12.
2022-07-14T11:33:48.736469Z qemu-system-x86_64: -device {"driver":"vhost-user-fs-pci","id":"fs0","chardev":"chr-vu-fs0","queue-size":1024,"tag":"daniel","bus":"pci.5","addr":"0x0"}: vhost_backend_init failed: Protocol error"
```

What does work, weirdly enough, is to run in some random directory `sudo mount -o bind / .` and then use said directory as your share point. With that I can view my home directory, sd card, and a USB drive.

**Comment by ImBearChild** [comment]
> Is there a way to change that and specify access to _some_ outside folders?

Instead of symlink, you can use [bindfs](https://bindfs.org/) to mount a directory to another location. Just mount it under your shared folder, and VM can access it. That works for me on my VM.
